stop-mycroft.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
similar comments on this file with `start-mycroft.sh` (fix throughout) as applicable